src/Glpi/Asset/Capacity/HasNetworkPortCapacity.php

```diff
@@ -58,6 +59,12 @@ public function getCloneRelations(): array
         ];
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("Has network ports (like ethernet and wlan). Usually reported by automatic inventory");
```

Comment:
If we plan to add a "can be included from inventory" in the future, maybe should we just not never add the information (that way, translations won't be affected).
```suggestion
        return __("Has network ports (like ethernet and wlan)");
```